include/kitepp/kite/order.hpp

```diff
@@ -0,0 +1,162 @@
+ * USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+#pragma once
+#pragma clang diagnostic ignored "-Wundefined-inline"
+
+#include <vector>
+
+#include "../kite.hpp"
+#include "../utils.hpp"
+
+namespace kiteconnect {
+inline string kite::placeOrder(const placeOrderParams& params) {
+    // required parameters
+    utils::http::Params bodyParams = {
+        { "exchange", params.exchange },
+        { "tradingsymbol", params.symbol },
+        { "transaction_type", params.transactionType },
+        { "quantity", std::to_string(params.quantity) },
+        { "product", params.product },
+        { "order_type", params.orderType },
+    };
+    // optional parameters
+    utils::addParam(bodyParams, params.price, "price");
+    utils::addParam(bodyParams, params.validity, "validity");
+    utils::addParam(bodyParams, params.disclosedQuantity, "disclosed_quantity");
+    utils::addParam(bodyParams, params.triggerPrice, "trigger_price");
+    utils::addParam(bodyParams, params.squareOff, "squareoff");
+    utils::addParam(bodyParams, params.stopLoss, "stoploss");
+    utils::addParam(bodyParams, params.trailingStopLoss, "trailing_stoploss");
+    utils::addParam(bodyParams, params.tag, "tag");
+
+    return callApi<string, utils::json::JsonObject, true>("order.place",
+        bodyParams, { params.variety }, [](utils::json::JsonObject& data) {
+            return utils::json::get<string>(data, "order_id");
+        });
+};
+
+inline string kite::modifyOrder(const modifyOrderParams& params) {
+    utils::http::Params bodyParams = {};
+    // optional parameters
+    utils::addParam(bodyParams, params.parentOrderId, "parent_order_id");
+    utils::addParam(bodyParams, params.quantity, "quantity");
+    utils::addParam(bodyParams, params.price, "price");
+    utils::addParam(bodyParams, params.orderType, "order_type");
+    utils::addParam(bodyParams, params.triggerPrice, "trigger_price");
+    utils::addParam(bodyParams, params.validity, "validity");
+    utils::addParam(bodyParams, params.disclosedQuantity, "disclosed_quantity");
+
+    return callApi<string, utils::json::JsonObject, true>("order.modify",
+        bodyParams, { params.variety, params.orderId },
+        [](utils::json::JsonObject& data) {
+            return utils::json::get<string>(data, "order_id");
+        });
+};
+
+inline string kite::cancelOrder(
+    const string& variety, const string& orderId, const string& parentOrderID) {
+    string endpoint;
+    utils::FmtArgs fmtArgs;
+    if (variety == "bo") {
+        endpoint = "order.cancel.bo";
+        fmtArgs = { variety, orderId, parentOrderID };
+    } else {
+        endpoint = "order.cancel";
+        fmtArgs = { variety, orderId };
+    };
+    return callApi<string, utils::json::JsonObject, true>(
+        endpoint, {}, fmtArgs, [](utils::json::JsonObject& data) {
+            return utils::json::get<string>(data, "order_id");
+        });
+};
+
+inline string kite::exitOrder(
```

Comment:
Is this for backward compatibility? If yes, since this is a major version change you could remove this alias function. And `exiting order` might confuse with `exiting position` as well, which is essentially placing a counter order.